di-ipv-cimit-stub/lambdas/get-contra-indicators/src/main/java/uk/gov/di/ipv/core/getcontraindicators/domain/ContraIndicatorItem.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.di.ipv.core.getcontraindicators.domain;
+
+import lombok.Data;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+
+@Data
+public class ContraIndicatorItem implements Comparable<ContraIndicatorItem> {
+    private final String userId;
+    private final String sortKey;
+    private final String iss;
+    private final String issuedAt;
+    private final String ci;
+    private final String ttl;
+    private final String documentId;
+
+    @Override
```

Comment:
Can lose this when the class no longer comparable

Reply:
done